[BUG] Cannot find module '../src/types'

Issue: jest-playwright-preset/types/global.d.ts:2:33 - error TS2307: Cannot find module '../types' or its corresponding type declarations.
release: next

Reason: We don't include the src directory, thats why the type can't get resolved.

See here: https://unpkg.com/browse/jest-playwright-preset@1.3.0-rc2/types/global.d.ts

Let's include d.ts from the src directory?

--- a/types/global.d.ts
+++ b/types/global.d.ts
@@ -1,5 +1,9 @@
 import type { Page, Browser, BrowserContext } from 'playwright-core'
-import type { SkipOption } from '../src/types'
+
+type SkipOption = {
+  browser: 'chromium' | 'firefox' | 'webkit'
+  device?: string | RegExp
+}
 
 interface JestPlaywright {
   skip: (skipOptions: SkipOption, callback: Function) => void

--- a/types/global.d.ts
+++ b/types/global.d.ts
@@ -1,7 +1,9 @@
 import type { Page, Browser, BrowserContext } from 'playwright-core'
 
+type BrowserType = 'chromium' | 'firefox' | 'webkit'
+
 type SkipOption = {
-  browser: 'chromium' | 'firefox' | 'webkit'
+  browser: BrowserType
   device?: string | RegExp
 }
 
@@ -65,7 +67,7 @@ interface JestPlaywright {
 }
 
 declare global {
-  const browserName: string
+  const browserName: BrowserType
   const deviceName: string | null
   const page: Page
   const browser: Browser

